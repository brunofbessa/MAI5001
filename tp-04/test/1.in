1 One
1 Two
1 Four
1 Five
0
