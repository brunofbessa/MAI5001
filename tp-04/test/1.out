One Um 3 5
Two Dois 2 2
Four Quatro 2 4
Five Cinco 3 3
